pkgs/applications/audio/non/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, pkgs, fetchgit}:
+
+stdenv.mkDerivation rec {
+  name = "non";
+  version = "2015-10-6";
+  src = fetchgit {
+    url = "git://github.com/original-male/non.git";
```

Comment:
Is this fork on github official? According to http://non.tuxfamily.org/wiki/Downloads it looks like git://git.tuxfamily.org/gitroot/non/non.git is the canonical repo.